includes/core/exceptions/class-invalid-request-parameter-exception.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Class Invalid_Request_Parameter_Exception
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Core\Exceptions;
+
+use WCPay\Exceptions\Base_Exception;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Exception for throwing an error when payment method can not be identified or used.
```

Comment:
```suggestion
 * Exception for throwing an error when an invalid parameter has been provided to a request.
```

The current message seems like a left-over.